src/test/unittest/ut_fh.h

```diff
@@ -14,19 +14,22 @@ struct FHandle;
 
 enum file_handle_type { FH_FD, FH_HANDLE };
 
-#define FH_ACCMODE	(3)
-#define FH_RDONLY	(0)
-#define FH_WRONLY	(1)
-#define FH_RDWR		(2)
+#define FH_ACCMODE	(7)
+#define FH_RDONLY	(1)
+#define FH_WRONLY	(2)
+#define FH_EXEC		(4)
+#define FH_RDWR		(FH_RDONLY | FH_WRONLY)
+#define FH_EXEC_READ	(FH_EXEC | FH_RDONLY)
+#define FH_EXEC_WRITE	(FH_EXEC | FH_WRONLY)
```

Comment:
Please don't add constants for combinations of other flags, because it's easy to miss something... e.g. here you didn't add rwx combination.